### PR TITLE
Freedom wall

### PR DESCRIPTION
**Pull request by Tapispisan-YvesJohn**
fix ui and add FormsModule

### CODE REVIEW

File: orgbee/src/app/pages/freedom-wall/freedom-wall.component.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  cursor: pointer;
+  width: 50%;
+  border-radius: 10px;
+}
+
+.manage-wall-btn {
+  background-color: gray;
+}
+
+
```
Comment (Ternal-AprilJoy) [inline orgbee/src/app/pages/freedom-wall/freedom-wall.component.css]: Remove excess space.

File: orgbee/src/app/pages/freedom-wall/freedom-wall.component.html [hunk 2]
```diff
@@ -1 +1,36 @@
...
+    <span class="close-btn" (click)="showModal = false">&times;</span>
+    <textarea [(ngModel)]="newPostText" 
+        placeholder="Write your post here...">
+    </textarea>
+    <button (click)="addPost(); showModal = false">Add Post</button>
+  </div>
+</div>
+
+<ngx-masonry class="masonry-container">
+  <ngx-masonry-item *ngFor="let post of posts">
```
Comment (Ternal-AprilJoy) [inline orgbee/src/app/pages/freedom-wall/freedom-wall.component.html]: Use @for instead.

File: orgbee/src/app/pages/freedom-wall/freedom-wall.component.html [hunk 3]
```diff
@@ -1 +1,36 @@
...
+    <div class="masonry-item" [ngStyle]="{'background-color': post.color}"
```
Comment (Ternal-AprilJoy) [inline orgbee/src/app/pages/freedom-wall/freedom-wall.component.html]: Never use NGstyle put it in css instead.

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_0

File: orgbee/src/app/pages/freedom-wall/freedom-wall.component.ts [hunk 4]
```diff
@@ -1,10 +1,49 @@
...
 @Component({
   selector: 'app-freedom-wall',
   templateUrl: './freedom-wall.component.html',
   styleUrl: './freedom-wall.component.css'
 })
 export class FreedomWallComponent {
+  posts: any[] = [];
+  newPostText: string = '';
+  showPostCreation: boolean = false;
+  showModal: boolean = false;
```
Comment (Ternal-AprilJoy) [inline orgbee/src/app/pages/freedom-wall/freedom-wall.component.ts]: Dont use any type. I think you can write it like this.


```suggestion
   posts: Post[] = [];
  newPostText: string = '';
  showPostCreation: boolean = false;
  showModal: boolean = false;
```

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g26feb3ebb2b_0_0

File: orgbee/src/app/post-dialog/post-dialog.component.html [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<div class="dialog-content" [ngStyle]="{ 'background-color': data.backgroundColor }">
```
Comment (Ternal-AprilJoy) [inline orgbee/src/app/post-dialog/post-dialog.component.html]: Dont use ngStyle. Kindly write your styling in the css file.

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_0

File: orgbee/src/app/post-dialog/post-dialog.component.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import { Component, Inject, ElementRef, AfterViewInit } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-post-dialog',
+  templateUrl: './post-dialog.component.html',
+  styleUrls: ['./post-dialog.component.css']
+})
+export class PostDialogComponent {
+  constructor(@Inject(MAT_DIALOG_DATA) public data: any, private elementRef: ElementRef){}
```
Comment (Ternal-AprilJoy) [inline orgbee/src/app/post-dialog/post-dialog.component.ts]: Kindly reconstruct this code and not use 'any' type.
reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2c15720ff99_0_64

**Comment by Tapispisan-YvesJohn** [comment]
I apologize for the delay. I will submit the pull request after finalizing my code. Thank you for your patience.

**Comment by Ternal-AprilJoy** [comment]
This pull request will be closed since there will be a new branch where these changes will be merged.

**Pull request closed without merging**